recipes/delly/meta.yaml

```diff
@@ -1,15 +1,20 @@
 package:
   name: delly
-  version: '0.7.2'
+  version: '0.7.6'
 
 source:
-  git_url: https://github.com/tobiasrausch/delly.git
-  git_rev: v0.7.2
+  fn: delly_v0.7.6_linux_x86_64bit
+  url: https://github.com/tobiasrausch/delly/releases/download/v0.7.6/delly_v0.7.6_linux_x86_64bit
```

Comment:
Has the binary been built on CentOS <= 6, or is it including a static copy of libc? If not, we would need to rebuild here, because that is the minimal libc version supported by bioconda.